pkgs/servers/dict/gcide/default.nix

```
@@ -0,0 +1,106 @@
+{
+  autoreconfHook,
+  bison,
+  dict,
+  fetchurl,
+  flex,
+  lib,
+  libmaa,
+  stdenv,
+}:
+
+let
+  sed-scripts = {
+    check = ./sed-scripts/check.sed;
+    fixes = ./sed-scripts/fixes.sed;
+    post_webfilter = ./sed-scripts/post_webfilter.sed;
+  };
+
+  _major_debver = "0.48";
+  _debver = "${_major_debver}.5+nmu4";
+in
+stdenv.mkDerivation rec {
+  version = "0.54";
+
+  pname = "dict-gcide";
+  name = pname;
+  dbName = pname;
+
+  sourceRoot = "./${pname}";
+  srcs = [
+    (fetchurl {
+      url = "https://deb.debian.org/debian/pool/main/d/dict-gcide/dict-gcide_${_debver}.tar.xz";
+      hash = "sha256-45ITB4SnyRycjchdseFP5J+XhZfp6J2Dm5q+DJ/N4A4=";
+    })
+    (fetchurl {
+      url = "https://ftp.gnu.org/gnu/gcide/gcide-${version}.tar.xz";
+      hash = "sha256-IkFvbzYXWxYNw4i3VHUSUU1GRHPPfXyJjXOO+ybFHUI=";
+    })
+  ];
+
+  buildInputs = [
+    libmaa
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+    bison
+    dict
+    flex
+  ];
+
+  prePatch = ''
+    sed -Ei "/The Collaborative International Dictionary of English v.${_major_debver}/ {
+      s/${_major_debver}/${version}/ ;
+    }" scan.l;
+  '';
+
+  buildPhase = ''
+    make;
+
+    ## Do the conversion explicitly, instead of `make db', to account for all
+    ## the differences to the original build process.
+    ## LANG=C is required so that the index file is properly sorted.
+    sed -Ef "${sed-scripts.fixes}" ../gcide-${version}/CIDE.?  |
+      sed -f debian/sedfile |
+      ./webfilter |
+      sed -Ef "${sed-scripts.post_webfilter}" |
+      tee pre_webfmt.data |
+      LANG=C ./webfmt -c;
```

```suggestion
      LANG=C ./webfmt -c
```
the ; are not required